src/deps.ts

```diff
@@ -1,3 +1,5 @@
+import ProgressBar from "https://x.nest.land/progress@1.1.0/mod.ts";
```

Comment:
you can simply replace `import` with `export`

Reply:
i know, but thats not working idk why